Meta/build-root-filesystem.sh

```diff
@@ -163,6 +163,7 @@ ln -s ProfileViewer mnt/bin/pv
 ln -s WebServer mnt/bin/ws
 ln -s Solitaire mnt/bin/sl
 ln -s WebView mnt/bin/wv
+ln -s test mnt/bin/\[
```

Comment:
```suggestion
ln -s test mnt/bin/[
```
We don't need a backslash here.

Reply:
> [ is a shell builtin, not a special character like (, ), ;, &, |, etc.
> You don't need to escape builtins: echo alias works fine, you don't need to use echo \alias

I was talking about this change @bugaevc .

Reply:
Yeah, here you don't need a backslash of course, `[` is just a character like any other; it's just that some shells have a builtin named `[`